packages/genesys-spark-components/src/components/stable/gux-table-toolbar/gux-table-toolbar.tsx

```diff
@@ -139,7 +140,7 @@ export class GuxTableToolbar {
   }
 
   get allFilterContextual(): HTMLGuxTableToolbarCustomActionElement[] | null {
-    return this.filterActions?.concat(this.contextualActions);
+    return this.filterActions?.concat(this.contextualActions) ?? [];
```

Comment:
would this possibly concat a null value making the array at least `[null]`? Is that caught by `??`?

Reply:
Updated to a more straightforward approach 👍